backend/tracim_backend/tests/functional/test_webdav.py

```diff
@@ -157,6 +159,56 @@ def test_functional__webdav_access_to_workspace__nominal_case(self) -> None:
         )
         res = self.testapp.get('/test', status=200)
 
+    def test_functional__webdav_access_to_workspace__special_characters(self) -> None:
+        # verify initial state, workspace should not exist yet
+        dbsession = get_tm_session(self.session_factory, transaction.manager)
+        admin = dbsession.query(User) \
+            .filter(User.email == 'admin@admin.admin') \
+            .one()
+        uapi = UserApi(
+            current_user=admin,
+            session=dbsession,
+            config=self.app_config,
+        )
+        gapi = GroupApi(
+            current_user=admin,
+            session=dbsession,
+            config=self.app_config,
+        )
+        groups = [gapi.get_one_with_name('users')]
+        user = uapi.create_user('test@test.test', password='test@test.test',
+                                do_save=True, do_notify=False,
+                                groups=groups)  # nopep8
+        self.testapp.authorization = (
+            'Basic',
+            (
+                'test@test.test',
+                'test@test.test'
+            )
+        )
+        self.testapp.get('/%E2%A7%B8%CA%94%E2%A7%B9%23%E2%88%97', status=404)
+
+        # creating workspace
+        workspace_api = WorkspaceApi(
+            current_user=admin,
+            session=dbsession,
+            config=self.app_config,
+            show_deleted=True,
+        )
+        workspace = workspace_api.create_workspace('/?\\#*', save_now=True)  # nopep8
+        rapi = RoleApi(
+            current_user=admin,
+            session=dbsession,
+            config=self.app_config,
+        )
+        rapi.create_one(user, workspace, UserRoleInWorkspace.READER,
+                        False)  # nopep8
+        transaction.commit()
+
+        # /?\# -> /⧸ʔ⧹#∗
+        # urlencoded version
+        self.testapp.get('/%E2%A7%B8%CA%94%E2%A7%B9%23%E2%88%97', status=200)
```

Comment:
if this test case fails in the future, it's quite impossible to understand why it fails. Please add a comment explaining what path you are looking for.

Reply:
i add a comment about meaning of %E2%A7%B8%CA%94%E2%A7%B9%23%E2%88%97